### ISSUE TITLE
Rebuild logos

### ISSUE THREAD

**Issue by patrickdeutsch-wk** [plantuml-stdlib/gilbarbara-plantuml-sprites#3]
Hello,
I tried to run the build in hopes to compile newly added logos. I ran into issues getting the build to work with groovy. Would it be possible to setup automated releases with github actions to keep logos current? Or just do a refresh?

**Comment by Potherca** [comment]
Either one would be awesome. Any help would be greatly appreciated, as there is not much energy/effort available to maintain and improve this repo 😞

**Comment by patrickdeutsch-wk** [comment]
Yeah, I don't have a lot a familiarity with groovy. But I imagine my issues are by not using the correct jdk version or potentially groovy version.  This is where i get blocked. 

```
❯ docker run --rm -v "$PWD":/home/groovy/scripts -w  /home/groovy/scripts groovy:jdk8 groovy svgsFolderUrl2plantUmlSprites.groovy https://github.com/gilbarbara/logos/tree/master/logos

Caught: java.lang.LinkageError: loader constraint violation: loader (instance of org/codehaus/groovy/tools/RootLoader) previously initiated loading for a different type with name "org/w3c/dom/UserDataHandler"
java.lang.LinkageError: loader constraint violation: loader (instance of org/codehaus/groovy/tools/RootLoader) previously initiated loading for a different type with name "org/w3c/dom/UserDataHandler"
        at svgsFolderUrl2plantUmlSprites.svg2Png(svgsFolderUrl2plantUmlSprites.groovy:120)
        at svgsFolderUrl2plantUmlSprites$_run_closure2.doCall(svgsFolderUrl2plantUmlSprites.groovy:60)
        at svgsFolderUrl2plantUmlSprites.run(svgsFolderUrl2plantUmlSprites.groovy:55)
```

**Comment by Potherca** [comment]
Yesterday, @rabelenda merged #5 which updates the dependencies.

![image](https://user-images.githubusercontent.com/195757/156735670-cd6699d6-cb4d-417e-b0df-7c3cdba4ba04.png)

And #4 which updates all the logos.

When I run the latest version, I get similar errors to #1 for 3 logos (`firefox.svg`, `tastejs.svg`, `waffle-icon.svg`) but other than that things seem to be working. 

@patrickdeutsch-wk Can you confirm/refute my dinfings?

**Comment by rabelenda** [comment]
Hello @patrickdeutsch-wk, 

I tried the docker command you mentioned and haven't faced the issue you describe. Conversion goes fine (with some errors due to batik not supporting some css property valyes). 

Additionally, when I run the script locally (without docker) I use java 8 and groovy 3, which are the same versions included in the docker image you are using.

Weird that you get the error with docker and I don't. Was the pull of the image for groovy recent? maybe you are using a different image than me?

The error you describe is usually generated by conflicts of groovy and the jvm jars, as far as I have experienced in the past.
